bears/general/AnnotationBear.py

```diff
@@ -0,0 +1,172 @@
+from coalib.parsing.StringProcessing.Core import (search_for,
+                                                  search_in_between,
+                                                  unescaped_search_in_between)
+from coalib.bearlib.languages.LanguageDefinition import LanguageDefinition
+from coalib.results.SourceRange import SourceRange
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.HiddenResult import HiddenResult
+
+
+class AnnotationBear(LocalBear):
+
+    def run(self, filename, file: str, language: str, language_family: str):
```

Comment:
`file: str` is invalid because we give a tuple of strings ?

Also `:param:` for language, family needed